### PR TITLE
Add manual smoke test plans for MS Office, Libre Office, Web Browsers, IDEs

### PR DESCRIPTION
**Pull request by seanbudd**
Part of https://github.com/nvaccess/nvda/discussions/16304

Creates smoke test plans for MS office, Libre Office, IDEs and Web browsers to help test against regressions when migrating to 64bit

### CODE REVIEW

File: tests/manual/applications/IDEs/jetBrains.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+### Expected Results
+
+* NVDA announces line numbers, code, and search results
+* Editing, navigation, and code assistance features are accessible
+
+## 4. Search and Navigation
+
+### Steps
+
+1. Use `ctrl+f` to search within the current file
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/jetBrains.md:83]: You already had us do that in Editor Accessibility step 3

File: tests/manual/applications/IDEs/jetBrains.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+1. Use `ctrl+shift+f` to search across the project
+1. Use `ctrl+shift+n` to find any file, class, or symbol
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/jetBrains.md:85]: You said `ctrl+shift+n` creates a new project in Project Operations step 1.

File: tests/manual/applications/IDEs/jetBrains.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+1. Use `ctrl+n` to find a class
+1. Use `ctrl+shift+n` to find a file
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/jetBrains.md:87]: You already had us use `ctrl+shift+n` to "find any file, class, or symbol"

File: tests/manual/applications/IDEs/jetBrains.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+* NVDA announces version control dialogs and actions
+* All version control operations are accessible
+
+## 7. Settings and Plugins
+
+### Steps
+
+1. Use `ctrl+alt+s` to open Settings/Preferences
+1. Navigate through settings categories using arrow keys
+1. Use `ctrl+alt+s` to open Plugins settings
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/jetBrains.md:135]: You already had us use `ctrl+alt+s` to open settings

File: tests/manual/applications/IDEs/jetBrains.md
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/jetBrains.md:1]: Not a JetBrains user, so some of these comments may be obvious to experienced JetBrains users

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+### Expected Results
+
+* NVDA announces window title and major UI regions
+* Navigation between tabs and documents is clear and consistent
+
+## 2. File Operations
+
+### Steps
+
+1. Use `ctrl+n` to create a new file
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:39]: You already had us do this in Application Launch and Window Navigation step 6

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+1. Type sample text
+1. Use `ctrl+s` to save, enter a filename, and confirm
+1. Use `ctrl+o` to open an existing file
+1. Use `ctrl+w` to close the current file
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:43]: Already done in Application Launch and Window Navigation

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+* NVDA announces file dialogs and file actions
+* File operations are accessible and confirmed by speech
+
+## 3. Editor Accessibility
+
+### Steps
+
+1. Type and navigate text in the editor using arrow keys
+1. Use `ctrl+home` and `ctrl+end` to jump to start/end of file
+1. Use `ctrl+upArrow` and `ctrl+downArrow` to scroll without moving cursor
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:58]: Note that doing so causes the current line to be reread if the cursor isn't forced to move by scrolling off screen

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+* NVDA announces line numbers, text, and cursor position
+* Editing and navigation are accessible
+
+## 4. Search and Replace
+
+### Steps
+
+1. Use `ctrl+f` to open the find dialog
+1. Enter a search term and use `f3` to find next
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:74]: Using `f3` with the find dialog open doesn't report anything, as the dialog remains open

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+1. Use `shift+f3` to find previous
+1. Use `ctrl+h` to open the replace dialog
+1. Enter search and replace terms
+1. Use `alt+a` to replace all
+1. Use `alt+r` to replace current
+1. Use `ctrl+f3` to find in files
+
+### Expected Results
+
+* NVDA announces search dialogs, results, and replacements
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:84]: Many of these operations don't cause announcements, but are still accessible

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+* All search and replace features are accessible
+
+## 5. Multiple Tabs and Documents
+
+### Steps
+
+1. Open multiple files to create tabs
+1. Use `ctrl+tab` to switch between tabs
+1. Use `ctrl+shift+tab` to switch in reverse order
+1. Use `ctrl+w` to close individual tabs
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:94]: Already done in Application Launch and Window Navigation

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+* NVDA announces tab names and switching between tabs
+* Multiple document management is accessible
+
+## 6. Plugins and Settings
+
+### Steps
+
+1. Use `alt+p` to open the Plugins menu
+1. Navigate through available plugins
+1. Use `f7` to access Settings
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:110]: Is this a default setting? I may have removed it, but it does not work in my copy of NPP

File: tests/manual/applications/IDEs/notepadPlusPlus.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+1. Navigate through settings categories
+1. Change a simple setting and verify it takes effect
+
+### Expected Results
+
+* NVDA announces plugin names and settings options
+* Plugin management and settings are accessible
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/notepadPlusPlus.md:117]: Worth noting that there are a lot of problems with mislabelled controls in NPP's settings UI

File: tests/manual/applications/IDEs/visualStudio.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+* Solution and project operations are accessible and confirmed by speech
+
+## 3. Code Editor Accessibility
+
+### Steps
+
+1. Type and navigate code in the editor using arrow keys
+1. Use `ctrl+home` and `ctrl+end` to jump to start/end of file
+1. Use `ctrl+f` to open the find dialog and search for text
+1. Use `ctrl+h` to open the replace dialog
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/visualStudio.md:59]: Has actually using text replacement intentionally been omitted?

File: tests/manual/applications/IDEs/visualStudio.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+### Expected Results
+
+* NVDA announces debugging state, breakpoints, and window focus
+* Stepping, breakpoints, and variable inspection are accessible
+
+## 5. Search and Navigation
+
+### Steps
+
+1. Use `ctrl+f` to search within the current file
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/visualStudio.md:92]: Already done in Code Editor Accessibility

File: tests/manual/applications/IDEs/visualStudio.md
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/visualStudio.md:1]: Not a Visual Studio user, so some of these questions/comments may be obvious to experienced users.

**Review by SaschaCowley — COMMENTED** [review]
Partial review

File: tests/manual/applications/IDEs/vsCodium.md [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+
+## 1. Application Launch and Window Navigation
+
+### Steps
+
+1. Launch VSCodium
+1. Verify NVDA announces the VSCodium window title
+1. Use `ctrl+shift+p` to open the command palette
+1. Use `ctrl+tab` to switch between open files/tabs
+1. Use `ctrl+b` to toggle the sidebar
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/vsCodium.md]: Not sure about in Codium, but in VS Code `Ctrl+B` doesn't announce anything, unless you're focused in the sidebar, in which case you'll hear something because the focus moves.

File: tests/manual/applications/IDEs/vsCodium.md [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+1. Use `f1` to open the command palette (alternative)
+1. Use `ctrl+shift+e` to focus the Explorer
+1. Use `ctrl+shift+f` to focus the Search panel
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/vsCodium.md]:
```suggestion
1. Use `ctrl+shift+f` to focus the Search panel
1. Use `f6` to navigate between sections of the window.
```

File: tests/manual/applications/IDEs/vsCodium.md [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+1. Use `ctrl+home` and `ctrl+end` to jump to start/end of file
+1. Use `ctrl+f` to open the find bar and search for text
+1. Use `ctrl+h` to open the replace bar
+1. Use `ctrl+z` and `ctrl+y` to undo/redo
+1. Use `ctrl+shift+\` to jump to matching bracket
+1. Use `alt+upArrow` and `alt+downArrow` to move lines up/down
+
+### Expected Results
+
+* NVDA announces line numbers, text, and search results
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/vsCodium.md]: I don't think we support reporting line numbers in Monaco

File: tests/manual/applications/IDEs/vsCodium.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+* NVDA announces extension names, details, and actions
+* Installing and managing extensions is accessible
+
+## 6. Terminal
+
+### Steps
+
+1. Use ``ctrl+` `` to open the integrated terminal
+1. Type commands and review output
+1. Use `ctrl+shift+\` to create a new terminal
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/vsCodium.md]:
```suggestion
1. Use ``ctrl+shift+` `` to create a new terminal
```

File: tests/manual/applications/IDEs/vsCodium.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+1. Use `ctrl+tab` to switch between terminal and editor
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/vsCodium.md]: Personally, I would not switch this way. I'd use `ctrl+1` to switch back to editor group 1.

File: tests/manual/applications/IDEs/vsCodium.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
...
+
+* NVDA announces extension names, details, and actions
+* Installing and managing extensions is accessible
+
+## 6. Terminal
+
+### Steps
+
+1. Use ``ctrl+` `` to open the integrated terminal
+1. Type commands and review output
```
Comment (SaschaCowley) [inline tests/manual/applications/IDEs/vsCodium.md]: Possibly worth noting that you can use `alt+f2` or `ctrl+upArrow` to enter accessible view to review terminal output, and `esc` or `ctrl+downArrow` (on the last line of output) to close it

File: tests/manual/applications/libreOffice.md [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+
+## Writer Smoke Tests
+
+### 1. Launch and Document Navigation
+
+#### Steps
+
+1. Launch Writer
+1. Verify NVDA announces the Writer window title
+1. Use `ctrl+n` to create a new document
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:24]: Not sure if it's just my settings, but you're already in a blank document when you open Writer

File: tests/manual/applications/libreOffice.md [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+1. Use `tab` to move through the toolbar and document area
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:25]: This will just insert tabs into the document

Comment (SaschaCowley) [reply tests/manual/applications/libreOffice.md:25]:
```suggestion
1. Use `f6` and `tab` to move through the toolbar and document area
```

File: tests/manual/applications/libreOffice.md [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+#### Expected Results
+
+* NVDA announces window title, toolbar, and document area
+* Navigation is smooth and consistent
+
+### 2. Basic Editing and Formatting
+
+#### Steps
+
+1. Type sample text
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:37]: Is it worth mentioning that you can get some sample text to work with by typing "DT" or "LOREM" followed by `f3`?

File: tests/manual/applications/libreOffice.md [hunk 24]
```diff
@@ -0,0 +1,142 @@
...
+* Editing and formatting are accessible
+
+### 3. Reviewing and Navigation
+
+#### Steps
+
+1. Use `f7` for spell check
+1. Use `ctrl+f` to open the find bar
+1. Use `ctrl+g` to go to a page/section
+1. Use `shift+f5` to return to last edit
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:55]: Need to save, close and reopen a document for this to work. Also doesn't seem like NVDA is aware that the caret moves in this case.

File: tests/manual/applications/libreOffice.md [hunk 25]
```diff
@@ -0,0 +1,142 @@
...
+#### Expected Results
+
+* NVDA announces formatting changes and file actions
+* Editing and formatting are accessible
+
+### 3. Reviewing and Navigation
+
+#### Steps
+
+1. Use `f7` for spell check
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:52]: Be aware of #17887

File: tests/manual/applications/libreOffice.md [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+1. Use `ctrl+f` to open the find bar
+1. Use `ctrl+g` to go to a page/section
+1. Use `shift+f5` to return to last edit
+
+#### Expected Results
+
+* Spell check and navigation features are accessible
+* NVDA announces review results and navigation targets
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:60]: This one doesn't really hold on 32 bit

File: tests/manual/applications/libreOffice.md [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+
+## Calc Smoke Tests
+
+### 1. Launch and Basic Navigation
+
+#### Steps
+
+1. Launch Calc
+1. Verify NVDA announces the Calc window title
+1. Use `ctrl+n` to create a new spreadsheet
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:72]: Again, Calc automatically opens to a new workbook

File: tests/manual/applications/libreOffice.md [hunk 28]
```diff
@@ -0,0 +1,142 @@
...
+1. Use arrow keys to move between cells
+1. Use `tab` to move across cells and toolbar
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:74]: Pressing `tab` will just move you across a row.
```suggestion
1. Use `f6` and `tab` to move across cells and toolbar
```

File: tests/manual/applications/libreOffice.md [hunk 29]
```diff
@@ -0,0 +1,142 @@
...
+
+1. Enter data in cells
+1. Use `ctrl+c` to copy, `ctrl+v` to paste, `ctrl+x` to cut
+1. Use `ctrl+z` and `ctrl+y` to undo/redo
+1. Use `ctrl+s` to save, `ctrl+o` to open a spreadsheet
+1. Use `alt+f4` to close
+
+#### Expected Results
+
+* NVDA announces data entry and clipboard actions
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:93]: It doesn't announce clipboard actions at present

File: tests/manual/applications/libreOffice.md [hunk 30]
```diff
@@ -0,0 +1,142 @@
...
+* File operations are accessible
+
+### 3. Formulas and Navigation
+
+#### Steps
+
+1. Enter a formula (e.g., `=sum(a1:a5)`)
+1. Use `f2` to edit a cell
+1. Use `ctrl+arrowKey` to jump to the edge of data regions
+1. Use `ctrl+space` to select a column, `shift+space` to select a row
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:103]: Only seems to be reported if there's already more than 1 cell selected

Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:103]: This should probably also include shift+arrows for selection

File: tests/manual/applications/libreOffice.md [hunk 31]
```diff
@@ -0,0 +1,142 @@
...
+
+#### Expected Results
+
+* NVDA announces formula entry and cell editing
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:107]: The auto suggestions when typing formulae don't seem to be accessible at present

File: tests/manual/applications/libreOffice.md [hunk 32]
```diff
@@ -0,0 +1,142 @@
...
+
+## Impress Smoke Tests
+
+### 1. Launch and Basic Navigation
+
+#### Steps
+
+1. Launch Impress
+1. Verify NVDA announces the Impress window title
+1. Use `ctrl+n` to create a new presentation
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:120]: When I first open LO Impress, I get a new slideshow wizard. Pressing `Ctrl+N` in this window has no effect.

File: tests/manual/applications/libreOffice.md [hunk 33]
```diff
@@ -0,0 +1,142 @@
...
+1. Use `tab` to move through the toolbar and slide area
+1. Use arrow keys to move between slides and within slide content
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:122]: Page up and down moves between slides. Arrow keys moves the selected object on the slide

File: tests/manual/applications/libreOffice.md [hunk 34]
```diff
@@ -0,0 +1,142 @@
...
+
+#### Expected Results
+
+* NVDA announces window title, toolbar, and slide area
+* Navigation is smooth and consistent
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:127]: Arguably not the case at present

File: tests/manual/applications/libreOffice.md [hunk 35]
```diff
@@ -0,0 +1,142 @@
...
+
+### 2. Slide Editing and Presentation
+
+#### Steps
+
+1. Type sample text on a slide
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:133]: I don't think you can type directly onto a slide; you have to type into a textbox

File: tests/manual/applications/libreOffice.md [hunk 36]
```diff
@@ -0,0 +1,142 @@
...
+1. Use `ctrl+m` to add a new slide
+1. Use `ctrl+s` to save, `ctrl+o` to open a presentation
+1. Use `f5` to start slideshow, `esc` to exit
+1. Use arrow keys to move between slides during slideshow
```
Comment (SaschaCowley) [inline tests/manual/applications/libreOffice.md:137]: Nothing is currently reported when presenting in LOI

File: tests/manual/applications/IDEs/vsCodium.md (hunk 15 above)
Comment (SaschaCowley) [reply tests/manual/applications/IDEs/vsCodium.md]: As of [VS Code 1.103.0](https://code.visualstudio.com/updates/v1_103#_side-bar-visibility-announcements) showing and hiding the primary and secondary sidebars is now announced.

File: tests/manual/applications/msOffice.md [hunk 37]
```diff
@@ -0,0 +1,198 @@
...
+
+## Word Smoke Tests
+
+### 1. Launch and Document Navigation
+
+#### Steps
+
+1. Launch Word
+1. Verify NVDA announces the Word window title
+1. Use `ctrl+n` to create a new document
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:80]: Behaviour of this depends on "Show the Start screen when this application starts"

* Enabled (default) - the backstage view is open when you first open Word; `ctrl+n` takes you to the "New" tab.
* Disabled - `ctrl+n` immediately creates and opens a new document.

File: tests/manual/applications/msOffice.md [hunk 38]
```diff
@@ -0,0 +1,198 @@
...
+1. Use `tab` to move through the ribbon and document area
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:81]: Pressing `tab` in the document will just insert tabs. Movement from section to section is done with `f6`

File: tests/manual/applications/msOffice.md [hunk 39]
```diff
@@ -0,0 +1,198 @@
...
+#### Expected Results
+
+* NVDA announces window title, ribbon, and document area
+* Navigation is smooth and consistent
+
+### 2. Basic Editing and Formatting
+
+#### Steps
+
+1. Type sample text
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:93]: Possibly noting that typing `=rand()` on its own line then pressing enter will insert some sample text.

File: tests/manual/applications/msOffice.md [hunk 40]
```diff
@@ -0,0 +1,198 @@
...
+
+## PowerPoint Smoke Tests
+
+### 1. Launch and Basic Navigation
+
+#### Steps
+
+1. Launch PowerPoint
+1. Verify NVDA announces the PowerPoint window title
+1. Use `ctrl+n` to create a new presentation
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:128]: Moves to new section of backstage view if open, otherwise creates and opens new presentation.

File: tests/manual/applications/msOffice.md [hunk 41]
```diff
@@ -0,0 +1,198 @@
...
+1. Use `tab` to move through the ribbon and slide area
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:129]: Within a slide, `tab` will just cycle through the objects on the slide. `f6` must be used to move between sections of the window.

File: tests/manual/applications/msOffice.md [hunk 42]
```diff
@@ -0,0 +1,198 @@
...
+1. Use arrow keys to move between slides and within slide content
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:130]: Behaviour of arrow keys is very context dependent.

* When on a slide but not focused on any objects, switches slides. I suggest using `pageUp` and `pageDown` to switch slides instead.
* When focused **on** an object, they move the object on the slide.
* When focus is **in** an object, it moves the text insertion point.

File: tests/manual/applications/msOffice.md [hunk 43]
```diff
@@ -0,0 +1,198 @@
...
+#### Expected Results
+
+* NVDA announces window title, ribbon, and slide area
+* Navigation is smooth and consistent
+
+### 2. Slide Editing and Presentation
+
+#### Steps
+
+1. Type sample text on a slide
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:141]: You can't type directly onto a slide. The only situations I've been able to get typing directly to work are:

* On a slide with a title shape with no text - text is inserted into the title shape
* If focus is **on** a textbox, typing will move into the textbox

I think this should be reworded to be a bit more specific. I.E. talk about tabbing to the title shape and typing into it.

File: tests/manual/applications/msOffice.md [hunk 44]
```diff
@@ -0,0 +1,198 @@
...
+1. Use `ctrl+m` to add a new slide
+1. Use `ctrl+s` to save, `ctrl+o` to open a presentation
+1. Use `f5` to start slideshow, `esc` to exit
+1. Use arrow keys to move between slides during slideshow
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:145]: Arrow keys don't move between slides with NVDA on; they review the contents of the slide. `pageUp` and `pageDown`, or `space` and `backspace`, or `n` and `p` should be used instead

File: tests/manual/applications/msOffice.md [hunk 45]
```diff
@@ -0,0 +1,198 @@
...
+
+#### Expected Results
+
+* NVDA announces slide content and transitions
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:149]: What do you mean "NVDA announces [...] transitions"?

File: tests/manual/applications/msOffice.md [hunk 46]
```diff
@@ -0,0 +1,198 @@
...
+
+## Excel Smoke Tests
+
+### 1. Launch and Basic Navigation
+
+#### Steps
+
+1. Launch Excel
+1. Verify NVDA announces the Excel window title
+1. Use `ctrl+n` to create a new workbook
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:162]: New page of backstage view is shown if same is open; creates new workbook otherwise.

File: tests/manual/applications/msOffice.md [hunk 47]
```diff
@@ -0,0 +1,198 @@
...
+1. Use arrow keys to move between cells
+1. Use `tab` to move across cells and ribbon
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:164]: `tab` only moves horizontally (across rows). `f6` should be used to move between window sections.

File: tests/manual/applications/msOffice.md [hunk 48]
```diff
@@ -0,0 +1,198 @@
...
+
+## Objective
+
+To ensure NVDA functions correctly with Microsoft Office applications (Outlook, Word, PowerPoint, Excel) across different UIA settings and typical user workflows.
+This plan focuses on basic navigation, document interaction, and accessibility of key features.
+
+## UIA Settings
+
+For Word/Outlook, and Excel toggle the relevant UIA settings in Advanced Preferences.
+Note: UIA support is evolving; NVDA should work optimally with the default UIA setting.
```
Comment (SaschaCowley) [inline tests/manual/applications/msOffice.md:11]: I'm not sure what this section is trying to communicate?

**Review by SaschaCowley — CHANGES_REQUESTED** [review]
Partial review

File: tests/manual/applications/webBrowsers.md [hunk 49]
```diff
@@ -0,0 +1,207 @@
...
+
+* [Download Firefox (Official)](https://www.mozilla.org/firefox/)
+  * Firefox 64-bit (latest stable)
+  * Firefox 32-bit (latest stable, if available)
+  * Firefox Nightly 64-bit (for early issue detection)
+
+### UIA
+
+For Chrome and Edge, toggle each option for use UIA in Chromium in Advanced Preferences.
+Note: UIA support is experimental, NVDA should work optimally with the default UIA setting.
```
Comment (SaschaCowley) [inline tests/manual/applications/webBrowsers.md:44]: What are you getting at with this section?

File: tests/manual/applications/webBrowsers.md [hunk 50]
```diff
@@ -0,0 +1,207 @@
...
+
+## Test Cases
+
+### 1. Browser Installation and Basic Launch
+
+#### Steps
+
+1. Launch the browser
+1. Verify NVDA announces the browser window title correctly
+1. Navigate to a simple HTML page (e.g., `about:blank`)
```
Comment (SaschaCowley) [inline tests/manual/applications/webBrowsers.md:81]: Perhaps <https://example.com/> would work better, since there's actually content there?

File: tests/manual/applications/webBrowsers.md [hunk 51]
```diff
@@ -0,0 +1,207 @@
...
+* Basic navigation works in browse mode
+
+### 2. Browse Mode Testing
+
+#### Steps
+
+1. Load a W3C landmarks sample page
+1. Use arrow keys to navigate through content
+1. Use `h`, `shift+h` to navigate headings
+1. Use `l`, `shift+l` to navigate links
```
Comment (SaschaCowley) [inline tests/manual/applications/webBrowsers.md:95]:
```suggestion
1. Use `k`, `shift+k` to navigate links
```

File: tests/manual/applications/webBrowsers.md [hunk 52]
```diff
@@ -0,0 +1,207 @@
...
+1. Use `d`, `shift+d` to navigate landmarks
+1. Use `f`, `shift+f` to navigate form fields
+1. Use `t`, `shift+t` to navigate tables
```
Comment (SaschaCowley) [inline tests/manual/applications/webBrowsers.md:98]: None of these pages seem to have any tables.

File: tests/manual/applications/webBrowsers.md [hunk 53]
```diff
@@ -0,0 +1,207 @@
...
+
+* NVDA handles errors gracefully
+* Performance remains acceptable
+
+### 6. Dynamic Content Update Testing
+
+#### Preconditions
+
+* Browser is running with a W3C sample or custom page that demonstrates dynamic content updates (e.g., ARIA live regions, notifications, chat messages, updating counters, etc.)
+* NVDA is running in browse mode and focus mode as appropriate
```
Comment (SaschaCowley) [inline tests/manual/applications/webBrowsers.md:161]: I'd probably add "Report dynamic content changes" needs to be on.

File: tests/manual/applications/webBrowsers.md [hunk 54]
```diff
@@ -0,0 +1,207 @@
...
+
+#### Steps
+
+1. Load a page with ARIA live regions (e.g., [ARIA Live Region Example](https://www.w3.org/WAI/ARIA/apg/example-index/live_regions/polite.html))
+1. Trigger a polite live region update (e.g., by clicking a button or waiting for an automatic update)
+1. Trigger an assertive live region update
+1. Load a page with dynamic notifications (e.g., ARIA alert role)
+1. Trigger a notification (e.g., form validation error, chat message, etc.)
+1. Load a page with dynamic content insertion/removal (e.g., adding/removing list items, updating counters)
```
Comment (SaschaCowley) [inline tests/manual/applications/webBrowsers.md:170]: Would probably help to get some example pages or links for these.

**Review by SaschaCowley — CHANGES_REQUESTED** [review]
I've now reviewed all manual smoke test plans.

File: tests/manual/applications/IDEs/vsCodium.md (hunk 15 above)
Comment (seanbudd) [reply tests/manual/applications/IDEs/vsCodium.md]: addressed in #19208

File: tests/manual/applications/IDEs/vsCodium.md (hunk 16 above)
Comment (seanbudd) [reply tests/manual/applications/IDEs/vsCodium.md]: addressed in #19208

File: tests/manual/applications/IDEs/vsCodium.md (hunk 19 above)
Comment (seanbudd) [reply tests/manual/applications/IDEs/vsCodium.md]: Could you explain how you switch between terminals efficiently. The commands suggested by the command pallet (alt/control+arrowKeys) seems ineffective

**Comment by seanbudd** [comment]
I've moved the VS Code manual test plan into a PR for system tests and manual tests: #19208

I plan to also create:
- A PR to add generic web browser testing, that tests the same scenarios against Firefox, Edge and Chrome
- A PR to add Visual Studio testing, similar to #19208

LibreOffice and Notepad++ testing should also be developed to avoid a regression surrounding https://github.com/nvaccess/nvda/pull/19204#issuecomment-3520835059

See also: #18762 

Manual test for plans MS Office and JetBrains will remain in this PR without automated testing
